test/testinput/bufr_ncep_lgycld_rrfs.yaml

```diff
@@ -0,0 +1,107 @@
+# (C) Copyright 2022 NOAA/NWS/NCEP/EMC
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+
+observations:
+  - obs space:
+      name: bufr
+
+      obsdatain: "./testinput/rap.t06z.lgycld.tm00.bufr_d"
+
+      exports:
+        variables:
+          # MetaData
+          longitude:
+            query: "*/CLONH"
+            transforms:
+              - offset: 360.0
```

Comment:
Does this mean that CLONH data is -360 to 0, and it is transformed to 0 +360 ?

Reply:
@nicholasesposito, All values are negative, -80 means 80 oW, if I use 360+(-80) =280, this is correct.  This data for US-alike region.

Reply:
Ok sounds good.